native/NativeSampleApps/SmartSyncExplorer/src/com/salesforce/samples/smartsyncexplorer/ui/MainActivity.java

```diff
@@ -132,6 +132,15 @@ protected void onCreate(Bundle savedInstanceState) {
 	}
 
 	@Override
+	protected void logoutCompleteActions() {
+		super.logoutCompleteActions();
```

Comment:
Yes - I think that's a good idea to move it to the super classes. We also need to check if we should add this to `SalesforceDroidGapActivity` - it inherits from `CordovaActivity` instead of ours.